agent/src/agent/cli/prompt/pipeline/elastic.py

```diff
@@ -29,4 +29,8 @@ def prompt_query(self):
             errors = elastic_query.get_errors(query, offset_field)
             if errors:
                 raise click.ClickException(errors)
-            self.config['query'] = query
+
+            # todo duplicate
+            if 'override_source' not in self.config:
+                self.config['override_source'] = {}
```

Comment:
let's create it in the construct of PromptConfig and also remove from def prompt()